### PR TITLE
fix gunk convert panic when option contains []byte value

### LINKED ISSUE #245: gunk convert panic: string is not a valid value for uint8 for openapiv2_schema option
gunk convert panics when running with following valid proto file. 

Reproduce testscript:

```txt
gunk convert util.proto

-- util.proto --
syntax = "proto3";

package util;

message Util {
    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
	  example: {
	    value: '{ "status": "ok" }'
	  }
    };

    string hello = 1;
}
```

### PR DESCRIPTION
**Pull request by favadi**
Fix #245

### CODE REVIEW

File: reflectutil/setter.go [hunk 1]
```diff
@@ -155,6 +162,8 @@ func valueFor(typ reflect.Type, tag reflect.StructTag, value interface{}) reflec
 		valueStr = x.Sel.Name
 	case *protop.Literal:
 		valueStr = x.SourceRepresentation()
+	case uint8:
+		return reflect.ValueOf(x)
```
Comment (mvdan) [inline reflectutil/setter.go]: this case stands out a bit from the rest. add a comment?

File: reflectutil/setter.go [hunk 2]
```diff
@@ -132,6 +132,13 @@ func valueFor(typ reflect.Type, tag reflect.StructTag, value interface{}) reflec
 				list = reflect.Append(list, valueFor(typ.Elem(), tag, elt))
 			}
 		case *protop.Literal:
+			if typ.Elem().Kind() == reflect.TypeOf(byte(0)).Kind() && value.IsString {
```
Comment (mvdan) [inline reflectutil/setter.go]:
```
if etyp := typ.Elem(); etyp.Kind() == reflect.Uint8 && value.IsString {
```

(and reuse `etyp` below)

Comment (mvdan) [reply reflectutil/setter.go]: Or even better, declare `etyp` at the top of the `case reflect.Slice:`

File: reflectutil/setter.go (hunk 1 above)
Comment (favadi) [reply reflectutil/setter.go]: yeah, that doesn't look right, will trying to make it more obvious.

File: reflectutil/setter.go (hunk 2 above)
Comment (favadi) [reply reflectutil/setter.go]: > Or even better, declare etyp at the top of the case reflect.Slice:

I did that in latest commit.

File: reflectutil/setter.go (hunk 1 above)
Comment (favadi) [reply reflectutil/setter.go]: I removed this special case and return the whole slice of bytes instead.

**Pull request merged**